tools/cancellation.f90

```diff
@@ -0,0 +1,6 @@
+program cancellation
+      implicit none
```

Comment:
```suggestion
      implicit none
      
      integer(kind=4) :: i
```

Reply:
Hi Jon! One should be very careful here. Using "CFF" for commenting cards should be strictly avoided after having introded "CF" for "Cancellation Factors"